src/main/webapp/i18n/de/programmingExercise.json

```diff
@@ -71,9 +71,9 @@
                 "resetWeights": "Gewichtung zurücksetzen",
                 "weightsReset": "Die Gewichtung wurde erfolgreich zurückgesetzt.",
                 "weightsResetFailed": "Die Gewichtung konnte nicht zurückgesetzt werden.",
-                "saveWeights": "Gewichtung speichern",
-                "weightsUpdated": "Die Gewichtung wurde erfolgreich gespeichert.",
-                "weightCouldNotBeUpdated": "Die Gewichtung für den Testfall {{id}} konnte nicht übernommen werden: {{error}}."
+                "saveTestCases": "Testfälle speichern",
+                "testCasesUpdated": "Die Testfälle wurde erfolgreich gespeichert.",
```

Comment:
You forgot an "n" at the end of "wurde" ;)

Reply:
Du hast Recht ;)